diff --git a/build/Themes.astro b/build/Themes.astro
index 0994b46ea0d2a7bf2b9914e40b8982388d9aa50b..6d7b441fb12676ed6be024c25267b0c42c3e594e 100644
--- a/build/Themes.astro
+++ b/build/Themes.astro
@@ -12,6 +12,6 @@ export interface Props {
 const { defaultTheme = 'light' } = Astro.props as Props
 ---
 
-<script type="module" define:vars={{ defaultTheme }}>
+<script define:vars={{ defaultTheme }}>
   const n="theme",r=window.matchMedia("(prefers-color-scheme: dark)");function d(){return r.matches?"dark":defaultTheme}function a(e){const o=e!==void 0?e:localStorage.getItem(n);return o!=null?o:d()}function t(e){document.documentElement.setAttribute("data-theme",e),document.documentElement.style.colorScheme=e}function l(e){e.key===n&&t(a(e.newValue))}function m(){t(a())}function c(e){e.detail?(localStorage.setItem(n,e.detail),t(e.detail)):(localStorage.removeItem(n),t(a(e.detail)))}document.addEventListener("set-theme",c),window.addEventListener("storage",l),r.addEventListener("change",m),document.addEventListener("astro:after-swap",m),t(a());
 </script>
